content/release/ReleaseNotes365.md

```diff
@@ -0,0 +1,52 @@
+<!--
+title: "Contrast 3.6.5 - June 2019"
+description: "Contrast 3.6.5 June 2019"
+tags: "3.6.5 June Release Notes"
+-->
+
+## Fixes
+
+* Default library score settings weren't reflected correctly in the application score.
+* Attempting to approve or deny pending vulnerabilities caused an error.
+* Users with an Edit role couldn't export library data from the library's **Overview** page.
+* Sending a vulnerability to Jira with an attachment caused an error in some cases.
+* Managing user access groups caused internal server errors for some SaaS users.
+* Cache issues prevented users from seeing vulnerability details after applying a license to an application.
+* Slack integration notifications ignored New Asset selections made in **Notification Settings**.
+* The **Libraries** grid timed out and returned errors in some users' SaaS environments.
+
+## Improvements
+
+* Set up [session metadata](user-vulnerableapps.html#session) to pinpoint the source of vulnerabilities for each of your applications. To get started, configure your agent to report one of the available metadata types, including build numbers, branch names, repositories, and committers. Go to your application’s **Vulnerabilities** tab to see the data in the new vulnerabilities timeline, and use the **Seen By** column in the grid to filter vulnerabilities by specific values.
+
+* Use [source names](admin-ipmgmt.html#source-name) to label attack events by expected sources so you can promptly choose which attack events to investigate. All you need is the IP information for a known source (like a pen tester) to get started. When you view attacks in the **Attacks > Monitor** page and **Attacks Details** pages, Contrast will display the source name instead of the attacker’s IP information. (We’ll be adding source names to the Attack Events grid next!)
+
+* Contrast offers two new Protect rules against unsafe files: The Unsafe File Uploads rule blocks malicious files being uploaded to web applications, and the Zip File Overwrite rule protects against malicious files and directory structures within zip files uploaded to web applications. These rules available for all languages.
+
+
+## Agent Updates
+
+### Java summary
+
+Check back next release for updates!
+
+### .NET summary 
+
+The .NET team added support for agent use within [Docker containers](installation-netinstall.html#net-docker) for .NET 4.5.2+ applications. We improved accuracy of an insecure authentication protocol Assess rule, and added a new Assess rule to detect the “X-Powered-By” header. We fixed a bug that caused a null reference error in processing exclusions. We also added support for setting `application.code`.
```

Comment:
We need to add a note about removing support for two legacy configuration options. Here's my attempt:

> The legacy DotnetAgentService.exe.config is no longer supported for application pool whitelisting/blacklisting, please move these configuration values to `agent.dotnet.app_pool_whitelist` and `agent.dotnet.app_pool_blacklist` in contrast_security.yaml. This change applies to all versions after 19.5.4.